template.md

```diff
@@ -0,0 +1,321 @@
+
+```
+function fancyAlert(arg) {
+    if(arg) {
+        $.docs({div:'#foo'})
+    }
+}
+```
+
+### Code blocks with language identifier
+
+Use three backticks (&#96;&#96;&#96;) + a language ID to apply language-specific color coding to a code block. Here is the entire list of [GFM language IDs](https://github.com/jmm/gfm-lang-ids/wiki/GitHub-Flavored-Markdown-(GFM)-language-IDs).
+
+##### C&#9839;
+
+```c#
+using System;
+namespace HelloWorld
+{
+    class Hello 
+    {
+        static void Main() 
+        {
+            Console.WriteLine("Hello World!");
+
+            // Keep the console window open in debug mode.
+            Console.WriteLine("Press any key to exit.");
+            Console.ReadKey();
+        }
+    }
+}
+```
+#### Python
+
+```python
+friends = ['john', 'pat', 'gary', 'michael']
+for i, name in enumerate(friends):
+    print "iteration {iteration} is {name}".format(iteration=i, name=name)
+```
+#### PowerShell
+
+```powershell
+Clear-Host
+$Directory = "C:\Windows\"
+$Files = Get-Childitem $Directory -recurse -Include *.log `
+-ErrorAction SilentlyContinue
+```
+
+### Inline code
+
+Use backticks (&#96;) for `inline code`. Use inline code for command-line commands, database table and column names, and language keywords.
+
+## Blockquotes
+
+> The drought had lasted now for ten million years, and the reign of the terrible lizards had long since ended. Here on the Equator, in the continent which would one day be known as Africa, the battle for existence had reached a new climax of ferocity, and the victor was not yet in sight. In this barren and desiccated land, only the small or the swift or the fierce could flourish, or even hope to survive.
+
+## Images
+
+### Static Image or Animated gif
+
+![this is the alt text](./images/Logo_DotNet.png)
+
+### Linked Image
+
+[![alt text for linked image](./images/Logo_DotNet.png)](https://dot.net) 
+
+## Alerts
+
+### Note
+
+> [!NOTE]
+> This is NOTE
+
+### Warning
+
+> [!WARNING]
+> This is WARNING
+
+### Tip
+
+> [!TIP]
+> This is TIP
+
+### Important
+
+> [!IMPORTANT]
+> This is IMPORTANT
+
+## Videos
+
+### Channel 9
+
+<iframe src="https://channel9.msdn.com/Shows/On-NET/Shipping-NET-Core-RC2--Tools-Preview-1/player" width="960" height="540" allowFullScreen frameBorder="0"></iframe>
+
+### YouTube
+
+<iframe width="420" height="315" src="https://www.youtube.com/embed/g2a4W6Q7aRw" frameborder="0" allowfullscreen></iframe>
+
+## docs.ms extensions
+
+Docs.ms provides a few extensions to GitHub Flavored Markdown. 
+
+###  Includes
+
+You can embed the Markdown of one file into another using an include.
+
+[!INCLUDE[sample include file](./includes/sampleinclude.md)]
+
+### Buttons
+
+> [!div class="button"]
+[button links](./docs/core/index.md)
+
+### Selectors
+
+> [!div class="op_single_selector"]
+- [macOS](./docs/core/tutorials/using-on-macos.md)
+- [Windows](./docs/core/tutorials/using-on-windows.md)
+
+### Step-By-Steps
```

Comment:
not show what you mean. this would actually render at the place you define.

I think we need to file an issue for OPS too. Look like this renders on docs when placed at the top of the page: https://docs.microsoft.com/en-us/advanced-threat-analytics/deploy-use/install-ata-step2